webui/react/src/components/kit/LogViewer/LogViewer.tsx

```diff
@@ -533,7 +533,7 @@ const LogViewer: React.FC<Props> = ({
         </Tooltip>
         {handleCloseLogs && (
           <Link onClick={handleCloseLogs}>
-            <Icon aria-label="Close Logs" name="close" title="Close Logs" />
```

Comment:
We should support aria-labels for icons, in fact it should be required. I'd suggest making `title` required (for aria label) and using a `showTooltip: boolean` parameter if there are places where we don't want to show a tooltip.

Reply:
The names aren't all very descriptive though. What you're kind of suggesting is having `title` just be static (since title should always be the same as aria-label). If we're able to eliminate `title` entirely and just define it in the `Icon.tsx` folder that's an option.

Keep in mind that aria-label is what a screen reader will read out.

Reply:
Are there specific names that you think aren't descriptive enough? Maybe the solution could be to improve those handful of names (since most of them seem decent to me), rather than going through 100+ uses of <Icon /> to add titles or coming up with better aria-labels for the 70+ icons we provide.

Reply:
The fact that we've been doing a bad job isn't an excuse to continue doing a bad job here. A lot of these don't provide information on what they do: 'dai-logo', 'arrow-left', 'arrow-right', 'add-small', 'arrow-down', 'arrow-up', etc. Many of the icon names are _descriptions of the image_. An aria label should be a description of the what the icon _means_ in that context.

If an icon has a meaningful default title, we can set that here and not have to set it everywhere, you could do that for "undo" or "pause" but I don't know if you _can_ do that for all of them, you'd have to look into the usage.

Reply:
Alright, in that case I'll put this PR down for now and come back and do that once I'm done with the columns ticket.